deployment/cloud-run/unified_ai_api.py

```
@@ -0,0 +1,2228 @@
+    model_checks = {}
+
+    if emotion_detector is None:
+        health_status = "degraded"
+        issues.append("Emotion detection model not loaded")
+        model_checks["emotion_detection"] = {"status": "unavailable", "error": "Model not loaded"}
+    else:
+        try:
+            # Test emotion detection
+            emotion_detector.predict("I am happy today")
+            model_checks["emotion_detection"] = {"status": "healthy", "test_passed": True}
+        except Exception as exc:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)